lib/utils.js

```diff
@@ -63,6 +63,28 @@ const buildMarkdownText = (items, position) => {
   return markdownText.join('\n\n');
 };
 
+const buildPullRequestComment = (report) => {
+  const { assertions, items } = report;
+  var comment = `**Summary**:  
+  ${assertions.failed_count} **Tests Failed** ❌
+  ${assertions.skipped_count} **Tests Skipped** ⏩
+  ${items.length - (report.items.filter((item) => item.test_status == "FAIL").length) - assertions.skipped_count} / ${items.length} **Reqests Passed** ✅`;
+  let firstPass = true;
+  report.items.filter((item) => item.test_status == "FAIL")
+  .map((item) => {
+    if (firstPass == true) {
+      comment +=  " \n **The following tests are failing** \n";
+      firstPass = false;
+    }
+    comment += ( "\n [+] " + item.request_name + "\n");
+    item.failed.map((item) => comment += "  -  " + item + " \n");
```

Comment:
Rename this "item" variable since it should be "test"

Reply:
Done